arch/arm64/configs/aosp_lena_pdx213_defconfig

```diff
@@ -0,0 +1,814 @@
+CONFIG_LOCALVERSION="-perf"
+# CONFIG_LOCALVERSION_AUTO is not set
```

Comment:
We can't have nice things?

Reply:
are you really complaining about a first-ever defconfig used for testing before auto-config implementation in the config generator repo?

Reply:
We can, it's an omission from the stock config

Reply:
> are you really complaining about a first-ever defconfig used for testing before auto-config implementation in the config generator repo?

Why not use the config generator cleanly from the get-go instead of running around with incomplete defconfigs for half a year until someone is bothered enough?